src/lists/lists.macros.html

```diff
@@ -0,0 +1,34 @@
+{#
+  * @overview Macros for Lists.
+  * @module Lists
+  *
+#}
+
+{% from 'nightshade-core/src/links/links.macros.html' import link, link_icon %}
+
+{#
+  * Renders a list of links
```

Comment:
I'm following a convention to always refer to modules in title case, so any mention of the List module in the documentation should be capitalized. If y'all think it's overkill we can revise, but for now let's do that.